### PR TITLE
Readability for unreadable functions

### PR DESCRIPTION
**Pull request by ZenginU**
Can be considered similar to #1440 

Made eye load from constant,
I believe cumsum is much more readable, repeat has kind of duplicate code, made chunk more readable 
I do not think this is considered code golf, if so please inform me which parts

### CODE REVIEW

**Comment by tinyb0t** [comment]
Changes made in `tinygrad/`:
```
------------------------------------------------------------
files                             insertions       deletions
------------------------------------------------------------
tinygrad/tensor.py                         8              16
------------------------------------------------------------
lines added in the tinygrad folder: -8
```

File: tinygrad/tensor.py [hunk 1]
```diff
@@ -544,13 +538,13 @@ def tan(self): return self.sin() / self.cos()
 
   @staticmethod
   def _tri(r:int, c:int, k:int=0, **kwargs) -> Tensor: return Tensor.arange(r, **kwargs).unsqueeze(1).expand(r,c) <= Tensor.arange(-k, c-k, **kwargs).unsqueeze(0).expand(r,c)
-  def triu(self, k:int=0) -> Tensor: return Tensor._tri(self.shape[-2], self.shape[-1], k=k, dtype=self.dtype).where(self, Tensor.zeros_like(self))
-  def tril(self, k:int=0) -> Tensor: return Tensor._tri(self.shape[-2], self.shape[-1], k=k+1, dtype=self.dtype).where(Tensor.zeros_like(self), self)
+  def triu(self, k:int=0) -> Tensor: return Tensor._tri(self.shape[-2], self.shape[-1], k=k, dtype=self.dtype) * self
```
Comment (geohot) [inline tinygrad/tensor.py:541]: Err, there's issues with inf for this

Comment (ZenginU) [reply tinygrad/tensor.py:541]: Currently it supports inf but is it a bug? 
Tensor(float("inf")) * 0 = nan
Tensor([float("inf")]) * Tensor(0) = 0
Tensor([float("inf")]) * Tensor([0]) = nan
Can we trust this behaviour?

File: tinygrad/tensor.py [hunk 2]
```diff
@@ -544,13 +538,13 @@ def tan(self): return self.sin() / self.cos()
...
+  def tril(self, k:int=0) -> Tensor: return (1 - Tensor._tri(self.shape[-2], self.shape[-1], k=k+1, dtype=self.dtype)) * self
 
   # ***** math functions (unary) *****
   def trunc(self: Tensor) -> Tensor: return self.cast(dtypes.int32).contiguous().cast(self.dtype)
-  def ceil(self: Tensor) -> Tensor: return (self > (b := self.trunc())).where(b+1, b)
-  def floor(self: Tensor) -> Tensor: return (self < (b := self.trunc())).where(b-1, b)
+  def ceil(self: Tensor) -> Tensor: return (self > (b := self.trunc())) + b
```
Comment (geohot) [inline tinygrad/tensor.py:546]: I'd leave this too, dtypes shouldn't mix like this

File: tinygrad/tensor.py (hunk 1 above)
Comment (wozeparrot) [reply tinygrad/tensor.py:541]: i knew there was a reason it was where, where has better inf and nan handing than a masked multiple does

Comment (ZenginU) [reply tinygrad/tensor.py:541]: @wozeparrot So can we use .where for "def pad" as well. We rely on 1/inf = 0

**Pull request merged**